cross/squid/Makefile

```diff
@@ -13,15 +13,24 @@ LICENSE  =
 
 GNU_CONFIGURE = 1
 
-CONFIGURE_ARGS  = $(TC_CONFIGURE_ARGS) --prefix=$(INSTALL_PREFIX) --enable-icap-client --disable-strict-error-checking
+CONFIGURE_ARGS  = --disable-arch-native --enable-icap-client --disable-strict-error-checking
 
 HOSTCFGEN_NATIVE = $(WORK_DIR)/../../../native/$(PKG_NAME)/work-native/$(PKG_DIR)/src/cf_gen
-HOSTCFGEN = $(WORK_DIR)/$(PKG_DIR)/src/cf_gen
+HOSTCFGEN = $(WORK_DIR)/$(PKG_DIR)/cf_gen
 
 # Compile
+PRE_CONFIGURE_TARGET = myPreConfigure
 PRE_COMPILE_TARGET = myPreCompile
 
 include ../../mk/spksrc.cross-cc.mk
 
+.PHONY: myPreConfigure
+myPreConfigure:
+	mkdir -p $(WORK_DIR)/install/$(INSTALL_PREFIX)/include/security/
+	cp $(WORK_DIR)/install/$(INSTALL_PREFIX)/include/pam_appl.h $(WORK_DIR)/install/$(INSTALL_PREFIX)/include/security/
```

Comment:
There is no option in linux-pam to put the include files in a security subfolder? Or in squid to pick linux-pam in the root folder instead of searching in security?